protocol-designer/src/components/organisms/OverlayModal/LiquidsOverlayModal.tsx

```diff
@@ -0,0 +1,49 @@
+import { Dispatch, SetStateAction } from 'react'
```

Comment:
looks like you need to type these
```suggestion
import type { Dispatch, SetStateAction } from 'react'
```